spec/features/take_whole_pot_calculator_analytics_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.feature 'Take whole pot calculator analytics', type: :feature, js: true do
+  let(:pot) { '100000' }
+  let(:income) { '7000' }
+  let(:pension) { '5000' }
+  let(:pension_frequency) { 'annually' }
+  let(:pension_frequency_label) { 'Annual' }
+
+  def data_layer
+    Array(page.evaluate_script('window.dataLayer')).reduce({}, :merge)
+  end
+
+  scenario 'Calculator inputs available to the Google Tag Manage dataLayer' do
+    visit '/take-whole-pot'
+
+    fill_in 'pot', with: pot
+    fill_in 'income', with: income
+    fill_in 'pension', with: pension
+    select pension_frequency_label, from: 'pension_frequency'
+    click_on 'Calculate'
+
+    expect(data_layer).to include(
+      'pot' => pot,
```

Comment:
This will create invalid Javascript if the user input is a string.

These need to be an integer so that we don't have to clean up the input within GA. So if the user inputs "100,000.00" then in the `dataLayer` we expose `100000`.